src/components/TopBar/CallButton.vue

```diff
@@ -22,31 +23,40 @@
 				<VideoOutlineIcon v-else-if="silentCall" :size="20" />
 				<VideoIcon v-else :size="20" />
 			</template>
-			{{ startCallLabel }}
+			<template v-if="!isMobile">
+				{{ startCallLabel }}
+			</template>
```

Comment:
Probably this should be passed as a conditional scoped slot instead of always having a slot with conditional content to avoid some rendering issues in `NcActions`. But I need to re-check.
```suggestion
			<template v-if="!isMobile" #default>
				{{ startCallLabel }}
			</template>
```

Reply:
done